### PR TITLE
Move content from abstract to intro as suggested by Eliot

### PR DESCRIPTION
**Pull request by thomas-fossati**
${title}

### CODE REVIEW

**Comment by thomas-fossati** [comment]
I am going to merge this later tonight and submit -19 unless I hear you shout :-)

**Comment by adrianlshaw** [comment]
Thanks for the nudge Thomas. Should it say "IP semiconductor vendors and security consultancies"?

I noticed some inconsistency in plurality vs singular

Otherwise LGTM and thanks again

File: draft-tschofenig-rats-psa-token.md [hunk 1]
```diff
@@ -185,9 +180,18 @@ This document is produced through the Independent RFC Stream and was not subject
...
 
-Trusted execution environments are now present in many devices, which provide a
-safe environment to place security sensitive code such as cryptography, secure
-boot, secure storage, and other essential security functions. These security
+The Platform Security Architecture (PSA) is a set of hardware and firmware
+specifications, backed by reference implementations and a security
+certification program {{PSACertified}}.  The security specifications have been published by Arm,
+while the certification program and reference implementations are the result of
+a collaborative effort by companies from multiple sectors, including evaluation
+laboratories, semiconductor IP and security consultancy.  The main objective of
```
Comment (thomas-fossati) [inline draft-tschofenig-rats-psa-token.md:188]:
```suggestion
laboratories, IP semiconductor vendors and security consultancies.  The main objective of
```

**Comment by thomas-fossati** [comment]
> Thanks for the nudge Thomas. Should it say "IP semiconductor vendors and security consultancies"?
> 
> I noticed some inconsistency in plurality vs singular
> 
> Otherwise LGTM and thanks again

Good catch, thanks.  Now [613ec94](https://github.com/thomas-fossati/draft-psa-token/pull/124/commits/613ec943756c14fa9953c9a0b21cfc09a62a556a)

**Review by SimonFrost-Arm — APPROVED** [review]
LGTM

**Pull request merged**